third-party/install/uWebSockets.cmake

```diff
@@ -1,6 +1,6 @@
 macro(install_uWebSockets)
 
-    set(CMAKE_C_FLAGS "-O3 -DLIBUS_NO_SSL")
```

Comment:
should we not also be including the existing `CMAKE_C_FLAGS`?

Reply:
Probably; I will add that.

Reply:
Done. It currently has no effect, but I have tested that it works.

Reply:
it will have an effect if you manually specify flags when calling cmake though